app/src/main/java/org/oppia/app/topic/play/ChapterSummarySelector.kt

```diff
@@ -0,0 +1,8 @@
+package org.oppia.app.topic.play
+
+import org.oppia.app.model.ChapterSummary
+
+/** Interface to transfer the selected chapter summary to [TopicPlayFragmentPresenter]. */
+interface ChapterSummarySelector {
+  fun selectedChapterSummary(chapterSummary: ChapterSummary)
```

Comment:
Nit: ``selectChapterSummary`` since that's the action being taken.

Reply:
Done.